### PR TITLE
Align concepts covered in parent page

### PR DESCRIPTION
**Pull request by mohit-codecombat**
Aligned Shield images
Live-coding-classes-from-CodeCombat-

### CODE REVIEW

File: app/views/landing-pages/parents/PageParents.vue [hunk 1]
```diff
@@ -1145,7 +1144,7 @@ export default {
 .concept-covered-tab {
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
+  justify-content: end;
```
Comment (AndrewJakubowicz) [inline app/views/landing-pages/parents/PageParents.vue:1147]: [`end`  isn't recognized](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content#Support_in_Flex_layout) on many browser. Pair it with a `justify-content: flex-end` directly after for maximal browser support.

File: app/views/landing-pages/parents/PageParents.vue [hunk 2]
```diff
@@ -1179,6 +1178,16 @@ export default {
   margin-bottom: 20px;
 }
 
+.concept-covered-tab.beginner img{
+  max-width: 92px !important;
```
Comment (AndrewJakubowicz) [inline app/views/landing-pages/parents/PageParents.vue:1182]: The selector seems specific enough that `!important` shouldn't be required?

**Review by AndrewJakubowicz — CHANGES_REQUESTED** [review]
Looks fantastic to me!

Requesting changes for `flex-content` compatibility to support Chrome, and removing `!important`. I think it might just work, but the selector can also be made more specific if required.

File: app/views/landing-pages/parents/PageParents.vue (hunk 2 above)
Comment (mohit-codecombat) [reply app/views/landing-pages/parents/PageParents.vue]: Removed `!important` from here.

File: app/views/landing-pages/parents/PageParents.vue (hunk 1 above)
Comment (mohit-codecombat) [reply app/views/landing-pages/parents/PageParents.vue]: Replaced with `justify-content: flex-end`  thanks.

File: app/views/landing-pages/parents/PageParents.vue [hunk 3]
```diff
@@ -1179,6 +1178,16 @@ export default {
   margin-bottom: 20px;
 }
 
+.concept-covered-tab.beginner img{
+  max-width: 92px;
+}
+
+@media screen and (min-width: 700px) {
+  .concept-covered-tab.beginner .topics, .concept-covered-tab.intermediate .topics{
```
Comment (AndrewJakubowicz) [inline app/views/landing-pages/parents/PageParents.vue:1186]: You covered `.beginner` and `.intermediate` selector. Do we also need the third `.advanced` selector here?

**Review by AndrewJakubowicz — APPROVED** [review]
Small question, but looks good :D

File: app/views/landing-pages/parents/PageParents.vue (hunk 3 above)
Comment (mohit-codecombat) [reply app/views/landing-pages/parents/PageParents.vue:1186]: No we don't need .advanced as of now because we are adding gap between them.

**Pull request merged**